packages/office-ui-fabric-react/src/components/Calendar/Calendar.Props.ts

```diff
@@ -106,6 +106,12 @@ export interface ICalendarProps extends React.Props<Calendar> {
   navigationIcons?: ICalendarIconStrings;
 
   /**
+  * Whether the calendar should show the week number (weeks 1 to 53) before each week row
+  * @defaultvalue false
+  */
+  showWeekNumbers?: boolean;
```

Comment:
Should this prop be added also to the date picker component? Same question holds true for some of the other props we added recently. I know for instance we would want the showMonthPickerAsOverlay property in datePicker for compose scenarios in OWA.

Reply:
I will add a task!